docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst

```diff
@@ -13,8 +13,92 @@ Case definition in GBD 2017
 Neonatal sepsis and other neonatal infections are infections during the neonatal
 period that advance to a systemic bloodstream infection, the underlying cause of
 which can be meningitis, gastroenteritis, or other aetiologies. Neonatal
-pneumonia, however, is not included -- it is captured in our modelling of
+pneumonia, however, is not included -- it is captured in the modelling of
 pneumonia as a separate entity.
 
 .. todo::
-   Describe cause model
+   Add more information and references. In particular, find data about differences in global and 
+   gbd definitions, global prevalence and relation to folic acid during pregnancy.
+
+Modeling Measles in GBD 2017
+----------------------------
+
+It is worth noting that the ICD 10 and ICD 9 codes used in non-fatal and fatal GBD models are different. 
+The ICD 10 codes used in non fatal model are P36-P36.9, P38-P39.9. K29.0-K29.00, K29.1-K29.20, P77-P78.1 
+codes are also used in hospital/claims analyses of non fatal model The ICD 9 codes used are 771, 771.4-771.89. 
```

Comment:
```suggestion
are also used in hospital/claims analyses of non fatal model. The ICD 9 codes used are 771, 771.4-771.89. 
```
Minor edits for readability.

Reply:
Is 771==771.0? This is a question I also had for neonatal other. I've emailed the CoD team's PO to ask about that notation---though if that way of describing ranges is common to ICD codes and someone on our team is familiar with it, that would be helpful to know!